README.md

```diff
@@ -207,6 +214,12 @@ Typically, it is easier to get to work with `pdflatex` than PSTricks is.
 - German: [Neue TeX FAQ](https://texfragen.de) - A modern and updated LaTeX FAQ in German.
 - [BibTeX Style Examples](http://www.cs.stir.ac.uk/~kjt/software/latex/showbst.html) - Example output of common BibTeX styles (BST files).
 
+## Showcases
+
+- [Showcase of beautiful typography done in TeX & friends](https://tex.stackexchange.com/questions/1319/showcase-of-beautiful-typography-done-in-tex-friends) - A collection of showcases demonstrating the power of LaTeX.
```

Comment:
`collection of showcases` implies that there is a literal collection of several showcases, but this is only one showcase with multiple examples. Try `collection of examples` or `set of examples` instead.